.gitignore

```diff
@@ -1,3 +1,9 @@
+*.raw
```

Comment:
mb add a comment that these are all raw file extensions

Reply:
Yep